Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -631,9 +655,326 @@ def post_session_stats(self, hops=None):
         elif hasattr(self.ltsessions[hops], "post_session_stats"):
             self.ltsessions[hops].post_session_stats()
 
+    #####################################
+    # MOVED FROM LAUNCHMANY AND SESSION #
```

Comment:
Not sure why this is here. We can probably do without this.

Reply:
That's to help the reader understand that the following methods were not added here by design, but instead is legacy code. I would prefer it to stay here until the moment when we refactor Libtorren wrapper properly.

Reply:
Removed that